error in declaration files in the published NPM package

### Bug description

When installed from npm (tried with 3.0.1 and 3.0.2) and imported into a typescript project, the library causes the build to fail because the 'getConsent' function appears to be missing in the declaration files : 

```
> tsc -b .

../../node_modules/@inrupt/solid-client-access-grants/dist/index.d.ts:15:169 - error TS2305: Module '"./common/getters"' has no exported member 'getConsent'.

15 export { AccessGrantWrapper, getAccessModes, getExpirationDate, getId, getIssuanceDate, getIssuer, getRequestor, getResourceOwner, getResources, getTypes, getPurposes, getConsent, getCredentialSubject, getInbox, getInherit, } from "./common/getters";
                                                                                                                                                                           ~~~~~~~~~~

```

## Additional information

Looking at the `dist` files in the NPM package, that function is indeed required by `index.d.ts`, but missing from `./common/getters.d.ts` . It is in the source TS file, though.

Maybe it's ommitted by the bundler because of the `@internal` flag on the function ?
https://github.com/inrupt/solid-client-access-grants-js/blob/59482e438ca81b34445760a88bab9bbce7f94003/src/common/getters.ts#L128

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,10 @@ The following changes are pending, and will be applied on the next major release
 
 ## Unreleased
 
+### Bugfix
+
+- Internal function `getConsent` is no longer part of the exports, which was causing TS to not build in some setups.
+
 ## [3.0.2](https://github.com/inrupt/solid-client-access-grants-js/releases/tag/v3.0.2) - 2024-01-22
 
 ### Bugfix

--- a/src/common/index.test.ts
+++ b/src/common/index.test.ts
@@ -22,7 +22,6 @@
 import { describe, it, expect } from "@jest/globals";
 import {
   isValidAccessGrant,
-  AccessGrantWrapper,
   getAccessModes,
   getExpirationDate,
   getId,
@@ -39,7 +38,6 @@ describe("Index exports", () => {
   it("exposes expected things", () => {
     expect(Object.keys(packageExports)).toEqual([
       "isValidAccessGrant",
-      "AccessGrantWrapper",
       "getAccessModes",
       "getExpirationDate",
       "getId",
@@ -49,10 +47,13 @@ describe("Index exports", () => {
       "getResourceOwner",
       "getResources",
       "getTypes",
+      "getPurposes",
+      "getCredentialSubject",
+      "getInbox",
+      "getInherit",
     ]);
 
     expect(isValidAccessGrant).toBeDefined();
-    expect(AccessGrantWrapper).toBeDefined();
     expect(getAccessModes).toBeDefined();
     expect(getExpirationDate).toBeDefined();
     expect(getId).toBeDefined();

--- a/src/common/index.ts
+++ b/src/common/index.ts
@@ -22,8 +22,8 @@
 // This file is used to create a 'common' API object from the top-level exports.
 
 export { isValidAccessGrant } from "./verify/isValidAccessGrant";
+export type { AccessGrantWrapper } from "./getters";
 export {
-  AccessGrantWrapper,
   getAccessModes,
   getExpirationDate,
   getId,
@@ -33,4 +33,8 @@ export {
   getResourceOwner,
   getResources,
   getTypes,
+  getPurposes,
+  getCredentialSubject,
+  getInbox,
+  getInherit,
 } from "./getters";

--- a/src/index.test.ts
+++ b/src/index.test.ts
@@ -58,7 +58,6 @@ import {
   getResources,
   getTypes,
   getPurposes,
-  getConsent,
   getCredentialSubject,
   getInbox,
   getInherit,
@@ -92,7 +91,6 @@ describe("Index exports", () => {
     expect(saveSolidDatasetInContainer).toBeDefined();
     expect(GRANT_VC_URL_PARAM_NAME).toBeDefined();
     expect(gConsent).toBeDefined();
-    expect(AccessGrantWrapper).toBeDefined();
     expect(getAccessModes).toBeDefined();
     expect(getExpirationDate).toBeDefined();
     expect(getId).toBeDefined();
@@ -103,7 +101,6 @@ describe("Index exports", () => {
     expect(getResources).toBeDefined();
     expect(getTypes).toBeDefined();
     expect(getPurposes).toBeDefined();
-    expect(getConsent).toBeDefined();
     expect(getCredentialSubject).toBeDefined();
     expect(getInbox).toBeDefined();
     expect(getInherit).toBeDefined();

--- a/src/index.test.ts
+++ b/src/index.test.ts
@@ -47,7 +47,6 @@ import {
   saveSolidDatasetInContainer,
   GRANT_VC_URL_PARAM_NAME,
   gConsent,
-  AccessGrantWrapper,
   getAccessModes,
   getExpirationDate,
   getId,

--- a/src/index.ts
+++ b/src/index.ts
@@ -19,6 +19,10 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
+export type { DatasetWithId } from "@inrupt/solid-client-vc";
+
+export type { AccessGrantWrapper } from "./common/index";
+
 export type {
   AccessBaseOptions,
   AccessCredentialType,
@@ -36,6 +40,7 @@ export type { AccessModes } from "./type/AccessModes";
 export type { FetchOptions } from "./type/FetchOptions";
 export type { RedirectOptions } from "./type/RedirectOptions";
 export type { SaveInContainerOptions } from "./type/SaveInContainerOptions";
+
 export { CredentialIsAccessGrantAny } from "./type/AccessGrant";
 
 export {
@@ -79,7 +84,6 @@ export {
 } from "./resource";
 
 export {
-  AccessGrantWrapper,
   getAccessModes,
   getExpirationDate,
   getId,
@@ -90,10 +94,7 @@ export {
   getResources,
   getTypes,
   getPurposes,
-  getConsent,
   getCredentialSubject,
   getInbox,
   getInherit,
-} from "./common/getters";
-
-export type { DatasetWithId } from "@inrupt/solid-client-vc";
+} from "./common/index";

